File: single-language-analysis/dataset-files/java/TestSmell005.java
// Original URL: https://github.com/RachanSaini/java-testing-p1/blob/a41fdfa0c6ac3c2567ff44fbe5a57c41351ad8fd/LoopTest.java#L23-L28

public class TestSmell005 {
    @Test
    public void loop() {
        assertEquals("1223334444",loopObj.loop(4));
        assertEquals("1",loopObj.loop(1));
        assertEquals("122333",loopObj.loop(3));
    }
}

File: single-language-analysis/dataset-files/CSharp/CSharp.Tests/Tests/TestSmell005.cs
using Xunit;

public class TestSmell005
{
    [Fact]
    public void Loop()
    {
        Assert.Equal("1223334444", LoopObj.Loop(4));
        Assert.Equal("1", LoopObj.Loop(1));
        Assert.Equal("122333", LoopObj.Loop(3));
    }
}